File: site/static/templates/blank_play_state/java/core/src/main/java/{{PACKAGE_PATH}}/{{GAME}}Game.java
package {{PACKAGE}};

import org.flixelgdx.FlixelGame;

/**
 * Your game entry class.
 *
 * <p>FlixelGame owns the window settings and picks the first FlixelState.
 * 
 * <p>If you are new here, start in {@link PlayState}. That file is where you spawn sprites,
 * load sounds, and write your first update loop.
 */
public class {{GAME}}Game extends FlixelGame {

  public {{GAME}}Game() {
    super("{{GAME_NAME_ESC_JAVA}}", 640, 480, new PlayState());
  }
}

File: site/static/templates/blank_play_state/kotlin/core/src/main/kotlin/{{PACKAGE_PATH}}/{{GAME}}Game.kt
package {{PACKAGE}}

import org.flixelgdx.FlixelGame

/**
 * Your game entry class.
 *
 * FlixelGame owns the window settings and picks the first FlixelState.
 *
 * If you are new here, start in PlayState.kt. That file is where you spawn sprites,
 * load sounds, and write your first update loop.
 */
class {{GAME}}Game : FlixelGame("{{GAME_NAME_ESC}}", 640, 480, PlayState())
